chapters/jina_box/index.md

```diff
@@ -0,0 +1,178 @@
+## Jinabox
+
+##### Jinabox is a lightweight, customizable omnibox. You can use it for searching text, images, videos, audio or any kind of data with a Jina backend.
+
+![jina box](jinabox.gif)
+
+
+### Features
+
+- Easy-to-use web component for Jina search backend
+- Users can drag and drop multi-modal data as search queries
+- Webcam and microphone support (in progress, contributions welcome! ❤️)
+- Visualize search results in a dropdown list or separate container
+- Highly customizable
+
+### Before you start
+
+Jinabox is a frontend for Jina. You will need to spin up a Jina backend to get search results. Based on what you want to search, run one of our example Jina backends:
+
+- [Pokedex](https://github.com/jina-ai/examples/tree/master/pokedex-with-bit) (real image data): `docker run -p 65481:65481 -e "JINA_PORT=65481" jinaai/hub.app.bitsearch-pokedex search`
+- [Southpark](https://github.com/jina-ai/examples/tree/master/southpark-search) (real text data): `docker run -p 45678:45678 jinaai/hub.app.distilbert-southpark`
+- [MP4 mock](https://github.com/jina-ai/examples/tree/master/io-mock-app) (placeholder video data): `docker run -p 65481:65481 -e "JINA_PORT=65481" jinaai/hub.app.iomock.mp4 search`
+- [Wav mock](https://github.com/jina-ai/examples/tree/master/io-mock-app) (placeholder audio data): `docker run -p 65481:65481 -e "JINA_PORT=65481" jinaai/hub.app.iomock.sound search` 
+### Open jinabox.js in your Browser
+
+Just go to https://jina.ai/jinabox.js/ in your browser to open up the search interface.
+
+
+### Set Server Endpoint
+
+Based on the Docker port you are exposing (i.e. 65481 or 45678 in above Docker images), set jinabox's server endpoint to:
+
+    http://localhost:<port_number>/api/search
+
+
+![server_endpoint](server_endpoint.png)
+
+### Start Searching!
+
+Drag and drop images, video, or audio, or type in text to search.
+
+![jina box_search](jinabox_search.gif)
+
+
+## Jinabox in your Project
```

Comment:
Add a lead up sentence here to make it flow better. Something along the lines of 'this is how you can add a Jina Box to your existing web project'